setup.py

```diff
@@ -114,6 +114,7 @@
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
Django REST Framework 3.9 officially only [supports Python 3.7](https://github.com/encode/django-rest-framework/blob/b1c4d8b59b67084ed07a35ea9a35629805039aaa/setup.py#L75). It would be nice to bump dependencies before upgrading the Python.

Reply:
I thought pip will at least say something about it while preparing environment for tests.
This means I have to check every package in dependencies for python 3.9 support...